test/integration/puppetlabs/services/jruby/jruby_puppet_internal_test.clj

```diff
@@ -0,0 +1,34 @@
+(ns puppetlabs.services.jruby.jruby-puppet-internal-test
+  (:require [clojure.test :refer :all]
+            [puppetlabs.services.jruby.jruby-puppet-internal :as jruby-internal]
+            [puppetlabs.services.jruby.jruby-testutils :as jruby-testutils])
+  (:import (java.util.concurrent LinkedBlockingDeque)))
+
+  (deftest ^:integration settings-plumbed-into-jruby-container
+    (testing "setting plumbed into jruby container for"
```

Comment:
Hmmm, I think I wrote this?  :smile: I did it here as "setting plumbed into jruby container for" so that when the individual assertions would fail below that the result would be concatenated into the output, i.e., if the "connect timeout milliseconds" test's assertion were to fail, the testing string written to test output would be "setting plumbed into jruby container for http_connect_timeout_milliseconds".  Does that make sense or were you getting at something else?